README.md

```diff
@@ -79,6 +79,57 @@ see here for more BARs stub data https://github.com/hmrc/bank-account-reputation
 
 ---
 
+### MainTrans codes for SA
+In SA we look up the MainTrans code received in ChargeReference in the table below and display the corresponding charge
+type in the 'You Bill' page. If the code does not have a corresponding charge description, the user is take to the 
+generic sa ineligibility kick out page.
+
+| Main Trans code | English description                            | Welsh description                                   |
```

Comment:
it might be better to have this table sitting in confluence somewhere and linking to that. If you were looking for the rules confluence is probably where the wider audience would look maybe? Maybe see what Karen thinks to she if she thinks there's a good place for the table there